internal/handlers/common/error.go

```diff
@@ -119,3 +119,25 @@ func ProtocolError(err error) (*Error, bool) {
 var (
 	_ error = (*Error)(nil)
 )
+
+// formatBitwiseOperatorErr formats protocol error for given internal error and bitwise operator.
+// Mask value used in error message.
+func formatBitwiseOperatorErr(err error, operator string, maskValue any) error {
+	switch err {
+	case errNotWholeNumber:
+		return NewErrorMsg(ErrFailedToParse,
+			fmt.Sprintf("Expected an integer: %s: %#v", operator, maskValue),
+		)
```

Comment:
```suggestion
		return NewErrorMsg(
		    ErrFailedToParse,
			fmt.Sprintf("Expected an integer: %s: %#v", operator, maskValue),
		)
```

For the same reason we don't write

```go
func foo() { codeLine1
  codeLine2
}
```

The same in other places